### DISCUSSION TITLE
[plugin:vite:import-glob] Invalid glob: "@/**/*.{jsx,tsx}" (resolved: "@/**/*.{jsx,tsx}"). It must start with '/' or './'

### DISCUSSION THREAD

**Discussion by semoal** [shellscape/jsx-email#228] [General]
Migrating from 1.x to 2.x fails this, any guidance?

**Comment by shellscape** [comment]
Please make sure you have the latest version of all `@jsx-email` dependencies. That looks like something the old version of `app-preview` used.

**Reply by semoal** [reply]
<img width="930" alt="Screenshot 2024-10-23 at 18 38 14" src="https://github.com/user-attachments/assets/06edcf49-7655-4e8a-8417-de342e9cc4ac">
Looks like the @jsx-email/app-preview package is not updated on the 2.0.11 package... quite strange, gonna install it manually

**Reply by semoal** [reply]
`pnpm add @jsx-email/app-preview` correctly upgraded the package to 2.0.2 but it's strange that i've had to install it right? installing it correctly worked

**Reply by shellscape** [reply]
yeah it looks like maybe a package manager issue. I ran `npm info jsx-email --json` and it's definitely got the right peerDep version there:

```json
  "peerDependencies": {
    "react": "^18.2.0",
    "@jsx-email/app-preview": "^2.0.2",
    "@jsx-email/plugin-inline": "^1.0.1",
    "@jsx-email/plugin-minify": "^1.0.1",
    "@jsx-email/plugin-pretty": "^1.0.0"
  },
  ```
